src/main/java/uk/gov/hmcts/cmc/claimstore/services/DefendantResponseService.java

```diff
@@ -44,9 +44,16 @@ public Claim save(
             throw new ForbiddenActionException("Response for the claim " + claimId + " was already submitted");
         }
 
+        if (isCCJAlreadyRequested(claim)) {
+            throw new ForbiddenActionException(
```

Comment:
We could have more specific  `CCJAlreadyRequestedException ` exception subclassing from `ForbiddedActionException`.

Reply:
ok